### PR TITLE
Update transform inverter for postfix

### PR DESCRIPTION
**Pull request by Nic-Ma**
### Description
This PR updated the `TransformInverter` handler to align the postfix format with other components.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/handlers/transform_inverter.py [hunk 1]
```diff
@@ -104,7 +104,11 @@ def __call__(self, engine: Engine) -> None:
 
             transform_info = engine.state.batch[transform_key]
             if nearest_interp:
-                convert_inverse_interp_mode(trans_info=transform_info, mode="nearest", align_corners=None)
+                transform_info = convert_inverse_interp_mode(
+                    trans_info=deepcopy(transform_info),
```
Comment (Nic-Ma) [inline monai/handlers/transform_inverter.py:108]: Maybe several output_keys share the same batch_key during inverting, for example, all items use the transform_info of `label`, should deepcopy if we modify it.

**Review by wyli — APPROVED** [review]
thanks

File: monai/handlers/transform_inverter.py (hunk 1 above)
Comment (wyli) [reply monai/handlers/transform_inverter.py:109]: thanks, we agreed that we should create some followup test cases

**Pull request merged**